src/DotNetWorker.Grpc/RpcExtensions.cs

```diff
@@ -131,5 +135,26 @@ internal static FunctionDefinition ToFunctionDefinition(this FunctionLoadRequest
                 StackTrace = exception.StackTrace ?? string.Empty
             };
         }
+
+        /// <summary>
+        /// Returns an RpcException for when an exception is thrown by user's code. 
+        /// </summary>
+        /// <returns>An RpcException with IsUserException set to true.</returns>
+        internal static RpcException? ToUserRpcException(this Exception exception)
+        {
+            if (exception == null)
```

Comment:
super nit: I've noticed that in a lot of the worker code base we've switched to the [is operator](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/is - might be good to do the same here for consistency 

`if (exception is null)`

And if you want you can update `ToRpcException` too

Reply:
Yes, that is. the preference here.